components/EarlyAdoptersPage/index.jsx

```diff
@@ -0,0 +1,185 @@
+import { Octokit } from '@octokit/core';
+import Image from 'next/image';
+import PropTypes from 'prop-types';
+import { Fragment, useEffect, useState } from 'react';
+
+import {
+  SECTION_BOX_CLASS,
+  SUB_HEADER_CLASS,
+  TEXT_MEDIUM_LIGHT_CLASS,
+} from 'common-util/classes';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Button } from 'components/ui/button';
+
+const iconProps = { width: 24, height: 24 };
+const downloadLinks = [
+  {
+    id: 'darwin-arm64.dmg',
+    btnText: 'Download for Apple Silicon',
+    downloadLink: null,
+    icon: (
+      <Image
+        src="/images/operate-page/brand-apple.svg"
+        alt="Download for Apple Silicon"
+        {...iconProps}
+      />
+    ),
+  },
+  {
+    id: 'darwin-x64.dmg',
+    btnText: 'Download for MacOS Intel',
+    downloadLink: null,
+    icon: (
+      <Image
+        src="/images/operate-page/brand-apple.svg"
+        alt="Download for MacOS Intel"
+        {...iconProps}
+      />
+    ),
+  },
+  {
+    id: 'win32-x64.exe',
+    btnText: 'Download for Windows x64',
+    downloadLink: null,
+    icon: (
+      <Image
+        src="/images/operate-page/brand-windows.svg"
+        alt="Download for Windows x64"
+        {...iconProps}
+      />
+    ),
+  },
+];
+
+const octokit = new Octokit({
+  auth: process.env.NEXT_PUBLIC_GITHUB_AUTH_TOKEN,
+});
+
+async function getLatestRelease(agentType) {
+  try {
+    // Get all created tags
+    const tags = await octokit.request(
+      'GET /repos/valory-xyz/olas-operate-app/tags',
+      {
+        owner: 'valory-xyz',
+        repo: 'olas-operate-app',
+        headers: { 'X-GitHub-Api-Version': '2022-11-28' },
+      },
+    );
+
+    // Get all releases
+    const releases = await octokit.request(
+      'GET /repos/valory-xyz/olas-operate-app/releases',
```

Comment:
ideally I should've just requested only this, as each release inside has a tag, but for some reason needed releases have tags without `-modius` or `-agentsfun` postfixes >.<